marathon.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "hdfs-mesos",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mesosphere/hdfs-mesos:latest",
+      "network": "HOST"
+    }
+  },
+  "cpus": 0.1,
+  "mem": 256.0,
+  "instances": 1,
+  "env": {
+    "MESOS_HDFS_DATA_DIR": "/tmp/hdfs/data",
```

Comment:
Why should we need to manually set all these environment variables instead of falling back to the default values? I would think we would only need to specify any values that deviate from the defaults.